server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -97,7 +100,7 @@ public NewIndexType createLongField(String fieldName) {
     }
 
     public NewIndexType createDynamicNestedField(String fieldName) {
-      return setProperty(fieldName, ImmutableMap.of("type", "nested", "dynamic", "true"));
+      return setProperty(fieldName, ImmutableMap.of("type", "nested", "dynamic", "true", "include_in_parent", "true"));
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Define a constant instead of duplicating this literal "nested" 3 times.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)